python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py

```diff
@@ -425,7 +425,10 @@ def update_events_from_temp(db_manager: DatabaseManager) -> None:
     process_logger.log_complete()
 
 
-def process_gtfs_rt_files(db_manager: DatabaseManager) -> None:
+def process_gtfs_rt_files(
```

Comment:
we need both dbs now. one to figure out what files we have to extract and one to do the compute for the tranfromations.